src/Countries/US/NorthDakota/NorthDakotaIncome/V20200101/NorthDakotaIncome.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\NorthDakota\NorthDakotaIncome\V20200101;
+
+use Appleton\Taxes\Countries\US\NorthDakota\NorthDakotaIncome\NorthDakotaIncome as BaseNorthDakotaIncome;
+use Illuminate\Database\Eloquent\Collection;
+
+class NorthDakotaIncome extends BaseNorthDakotaIncome
+{
+    const SUPPLEMENTAL_TAX_RATE = 0;
+
+    const ANNUALLY = 4200;
+
+    const MARRIED_FILING_JOINTLY_NEW_STEP_2_NOT_CHECKED = [
+        [0, 0, 0],
+        [24440, 0.1, 0],
+        [43780, 0.12, 1940],
+        [103380, 0.22, 9091],
+        [192800, 0.24, 28767],
+        [345800, 0.32, 65496],
+        [432640, 0.35, 93252],
+        [636740, 0.37, 164700],
+    ];
+
+    const MARRIED_FILING_JOINTLY_NEW_STEP_2_IS_CHECKED = [
+        [0, 0, 0],
+        [12220, 0.1, 0],
+        [21890, 0.12, 967],
+        [51690, 0.22, 4542],
+        [96400, 0.24, 14381],
+        [172900, 0.32, 32739],
+        [216320, 0.35, 46634],
+        [318400, 0.37, 82360],
+    ];
+
+    const SINGLE_NEW_STEP_2_NOT_CHECKED = [
+        [0, 0, 0],
+        [12220, 0.1, 0],
+        [21890, 0.12, 967],
+        [51690, 0.22, 4542],
+        [96400, 0.24, 14381],
+        [172900, 0.32, 32739],
+        [216320, 0.35, 46634],
+        [522500, 0.37, 153795],
+    ];
+
+    const SINGLE_NEW_STEP_2_IS_CHECKED = [
+        [0, 0, 0],
+        [6080, 0.1, 0],
+        [10970, 0.12, 489],
+        [25840, 0.22, 2273],
+        [48200, 0.24, 7193],
+        [86500, 0.32, 16378],
+        [108160, 0.35, 23317],
+        [261250, 0.37, 76898],
+    ];
+
+    const HEAD_OF_HOUSEHOLD_NEW_STEP_2_NOT_CHECKED = [
+        [0, 0, 0],
+        [18360, 0.1, 0],
+        [32190, 0.12, 1383],
+        [71190, 0.22, 6063],
+        [102500, 0.24, 12962],
+        [179040, 0.32, 31320],
+        [222460, 0.35, 45214],
+        [528630, 0.37, 152376],
+    ];
+
+    const HEAD_OF_HOUSEHOLD_NEW_STEP_2_IS_CHECKED = [
+        [0, 0, 0],
+        [9150, 0.1, 0],
+        [16120, 0.12, 697],
+        [35620, 0.22, 3037],
+        [51270, 0.24, 6480],
+        [89540, 0.32, 15666],
+        [111230, 0.35, 22604],
+        [264320, 0.37, 76185],
+    ];
+
+    public function compute(Collection $tax_areas)
+    {
+        if ($this->isUserClaimingExemption()) {
+            return 0.00;
+        }
+
+        $tax_amount = $this->getAdjustedWageAmount();
+        $tax_amount = $this->getTentativeAmount($tax_amount);
+        $tax_amount -= $this->tax_information->dependents_deduction_amount;
+        $tax_amount = $tax_amount > 0 ? $tax_amount / $this->payroll->pay_periods : 0;
+        $tax_amount += $this->tax_information->extra_withholding;
```

Comment:
so the extra withholding should be divided among the pay periods?

Reply:
The tax amount is divided by the pay periods then the extra withholding is added to that.

Reply:
right, so is what they fill out on the form for extra withholding supposed to be for each pay period or is it supposed to be for the year?

Reply:
each pay period
<img width="493" alt="Screen Shot 2020-05-22 at 10 54 42 AM" src="https://user-images.githubusercontent.com/4420905/82685997-b7812b80-9c1a-11ea-8a33-98336c2d626b.png">

Reply:
just double checking

Reply:
I don't mind. I went to the form to ensure the correct answer. Double checking never hurts